### PR TITLE
Show image descriptions in gallery

### PR DESCRIPTION
**Pull request by charlag**
I made it so that gallery knows about real attachments. It will make things easier when we will add video there.
It shows description on semi-transparent background. When there's no description it is hidden. It is also synced with toolbar - you can press picture to hide both toolbar and description.

screenshot_20180505-033719

### CODE REVIEW

**Comment by connyduck** [comment]
Nice!
But do you know what also is a big complaint from users? That there is no way to see the toot when opening media from the media tab. Showing the toot next to the image description probably needs too much space 🤔 But what about an additional button in the toolbar next to the download button?

**Comment by charlag** [comment]
Yeah, I know, I was thinking about it but not sure I can do it now. We should wrap Attachment into something else which has statusId field so we know where this attachment belongs to. I will try later today. Do you want to include this into 1.8?

**Comment by charlag** [comment]
Okay, I've added the option but there are some problems with theming and I have no idea where are they coming from. If anyone could step in and fix it I would be *really* glad.

File: app/src/main/res/layout/activity_view_media.xml [hunk 1]
```diff
@@ -15,8 +15,6 @@
         android:id="@+id/toolbar"
         android:layout_width="match_parent"
         android:layout_height="?attr/actionBarSize"
-        android:theme="@style/AppTheme.Account.AppBarLayout"
-        app:popupTheme="@style/AppTheme.Account.ToolbarPopupTheme.Dark"
```
[the remark this answers is not in the document]
Comment (charlag) [reply app/src/main/res/layout/activity_view_media.xml]: Okay, thx, I'll try it!

**Comment by charlag** [comment]
I cnanot check it RN but there's nothing fishy on code so it should be good?

**Pull request merged**